docs/2-fork-accountability.md

```diff
@@ -0,0 +1,161 @@
+# Fork accountability
+
+As we know, the agreement property of the Tendermint consensus algorithm cannot be violated if there are at most *f* faulty processes in the system.
+What happen if more than *f* processes are faulty?
+
+A well-designed consensus protocol should provide some guarantees when this limit is exceeded. The most important guarantee is **fork accountability**, where the processes that caused the consensus to fail can be identified and punished according to the protocol specifications. 
```

Comment:
Put "caused" between "".

Reply:
Fixed. Related question: when I say something like "a faulty process leads to a fork", should I put leads between "" as well? I thought it's quite clear what this means from the context but it's not indeed a formal term to define this action.

Reply:
We should discuss this. I am not sure which level of formality we should use here. I guess Adi can help us with this decision.